### PR TITLE
Remove references to `.all`

### PR DESCRIPTION
**Pull request by lesderid**
See https://gitter.im/luckyframework/Lobby?at=5e650773a2ccef3232dd9ec0

### CODE REVIEW

File: src/actions/guides/database/querying-deleting.cr [hunk 1]
```diff
@@ -58,13 +58,11 @@ class Guides::Database::QueryingDeleting < GuideAction
     * `find(id)` - Returns the record with the primary key `id`. Raise `Avram::RecordNotFoundError` if no record is found.
     * `last` - Returns the last record. Raise `Avram::RecordNotFoundError` if no record is found.
     * `last?` - Returns the last record. Returns `nil` if no record is found.
-    * `all` - Returns an array of all the records.
 
     ```crystal
     first_user = UserQuery.first
     last_user = UserQuery.last
     specific_user = UserQuery.find(4)
-    all_users = UserQuery.all
```
Comment (paulcsmith) [inline src/actions/guides/database/querying-deleting.cr:67]: Could you add this back in, but call `new` instead? Just so people have it there if they need it

**Review by paulcsmith — COMMENTED** [review]
One small suggestion. Thanks for the PR!

**Review by jwoertink — APPROVED** [review]
Looks fine to me

**Review by paulcsmith — APPROVED** [review]
Looks great!

**Pull request merged**